src/contracts/pods/EigenPodManager.sol

```diff
@@ -651,29 +634,26 @@ contract EigenPodManager is
     }
 
     /**
-     * @notice Internal function to enter `podOwner` into undelegation limbo
+     * @notice Internal function to enter `podOwner` into undelegation limbo, and return the number of shares which were moved into undelegation limbo.
      * @dev Does nothing if the `podOwner` has no delegated shares (i.e. they are already in undelegation limbo or have no shares)
      * OR if they are not actively delegated to any operator.
+     *
+     * This method assumes the podOwner is delegated, as it's being called by the
+     * DelegationManager (and supplied with `delegatedTo`)
      */
-    function _enterUndelegationLimbo(address podOwner) internal {
-        if (!podOwnerHasNoDelegatedShares(podOwner) && delegationManager.isDelegated(podOwner)) {
-            // look up the address that the pod owner is currrently delegated to in EigenLayer
-            address delegatedAddress = delegationManager.delegatedTo(podOwner);
-
+    function _enterUndelegationLimbo(address podOwner, address delegatedTo) internal returns (uint) {
```

Comment:
uint256

Reply:
out of curiosity, why do you like the long form? i tend to use `uint` to keep things concise, since everyone knows it's 256.

i'm fine with being verbose if that's the style we're using. unified style is important :)

Reply:
fixed in https://github.com/Layr-Labs/eigenlayer-contracts/pull/183/commits/bf1de8a06817eb639c1c5a265b1aedfe0c9e1a33